spec/spec_helper.cr

```diff
@@ -1,12 +1,13 @@
 # NOTE: Constants should be set before require begins.
 
 ENV["AMBER_ENV"] = "test"
-TEST_PATH    = "spec/support/sample"
-PUBLIC_PATH  = TEST_PATH + "/public"
-VIEWS_PATH   = TEST_PATH + "/views"
-TESTING_APP  = "./tmp/test_app"
-APP_TPL_PATH = "./src/amber/cli/templates/app"
-CURRENT_DIR  = Dir.current
+TEST_PATH     = "spec/support/sample"
+PUBLIC_PATH   = TEST_PATH + "/public"
+VIEWS_PATH    = TEST_PATH + "/views"
+TEST_APP_NAME = "test_app"
```

Comment:
Why was this changed?

Reply:
We talked offline. 

This is to avoid the `split('/').last` on the test. See https://github.com/amberframework/amber/pull/315/files#diff-3269a1cd7605965238570f1b2aafc0a7L16